base/math.jl

```diff
@@ -102,6 +102,77 @@ macro horner(x, p...)
     return Expr(:block, :(local t = $(esc(x))), ex, :r)
 end
 
+"""
+    evalpoly(x, p::NTuple{N}) where {N}
```

Comment:
```suggestion
    evalpoly(x, p::Tuple)
```

Reply:
Blah. Good catches. I had two branches going and wasn't pushing all the changes.